zigbee: Configure frequency of FOTA image queries (#7590)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

      * Crypto library used for performing software AES ecryption.
        Now, the :ref:`nrfxlib:nrf_oberon_readme` is used instead of the Tinycrypt library.

  * :ref:`lib_zigbee_fota`:

    * Added:

>       * New :kconfig:option:`ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig option to configure interval

Reviewer: ```suggestion
      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig option to configure interval
```
?

Author: Fixed

>       * New :kconfig:option:`ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig optin to cinfigure interval
>         between queries for the avilable Zigbee FOTA images

Reviewer: ```suggestion
      * New :kconfig:option:`ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig option to configure interval
        between queries for the available Zigbee FOTA images.
```

Author: Fixed.

      * Crypto library used for performing software AES ecryption.
        Now, the :ref:`nrfxlib:nrf_oberon_readme` is used instead of the Tinycrypt library.

  * :ref:`lib_zigbee_fota`:

>     * Added:
> 
>       * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig option to configure interval
>         between queries for the available Zigbee FOTA images.
> 

Reviewer 2: ```suggestion
    * Added new :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_DISOVERY_INTERVAL_HRS` Kconfig option to configure the interval between queries for the available Zigbee FOTA images.
```

Author: Fixed
